src/components/PeopleTable.tsx

```diff
@@ -10,635 +118,123 @@ export const PeopleTable = () => {
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Name
-              <a href="#/people?sort=name">
+              <SearchLink params={sortClick('name')}>
                 <span className="icon">
-                  <i className="fas fa-sort" />
+                  <i
+                    className={classNames('fas', {
+                      'fa-sort': sort !== 'name',
+                      'fa-sort-up': sort === 'name' && !order,
+                      'fa-sort-down': sort === 'name' && order === 'desk',
+                    })}
+                  />
                 </span>
-              </a>
+              </SearchLink>
             </span>
           </th>
 
           <th>
             <span className="is-flex is-flex-wrap-nowrap">
               Sex
-              <a href="#/people?sort=sex">
+              <SearchLink params={sortClick('sex')}>
                 <span className="icon">
-                  <i className="fas fa-sort" />
+                  <i
+                    className={classNames('fas', {
+                      'fa-sort': sort !== 'sex',
+                      'fa-sort-up': sort === 'sex' && !order,
+                      'fa-sort-down': sort === 'sex' && order === 'desk',
```

Comment:
The `order` value 'desk' seems to be a typo. It should likely be 'desc' for descending order.